From c7a575b906897925b35ee76bb51ba43151f35d4e Mon Sep 17 00:00:00 2001
From: xiaoleGun <1592501605@qq.com>
Date: Tue, 26 Jul 2022 17:29:51 +0800
Subject: [PATCH 3/9] Revert "udfps: Add support for udfps on aod without
 having dedicated sensor"

This reverts commit 2a037995f50fb4bbb09309b1aecc5872dd25178c.

Signed-off-by: xiaoleGun <1592501605@qq.com>
---
 .../fingerprint/IUdfpsOverlayController.aidl        |  3 ---
 .../systemui/biometrics/UdfpsController.java        | 13 -------------
 .../hidl/FingerprintAuthenticationClient.java       |  9 ---------
 3 files changed, 25 deletions(-)

diff --git a/core/java/android/hardware/fingerprint/IUdfpsOverlayController.aidl b/core/java/android/hardware/fingerprint/IUdfpsOverlayController.aidl
index f5d9a76e50e3..648edda62171 100644
--- a/core/java/android/hardware/fingerprint/IUdfpsOverlayController.aidl
+++ b/core/java/android/hardware/fingerprint/IUdfpsOverlayController.aidl
@@ -40,7 +40,4 @@ oneway interface IUdfpsOverlayController {
 
     // Shows debug messages on the UDFPS overlay.
     void setDebugMessage(int sensorId, String message);
-
-    // Useful for Legacy devices not having dedicated aod sensor
-    void onAcquired(int sensorId, int acquiredInfo, int vendorCode);
 }
diff --git a/packages/SystemUI/src/com/android/systemui/biometrics/UdfpsController.java b/packages/SystemUI/src/com/android/systemui/biometrics/UdfpsController.java
index 438b12847b0d..8533f356d67a 100644
--- a/packages/SystemUI/src/com/android/systemui/biometrics/UdfpsController.java
+++ b/packages/SystemUI/src/com/android/systemui/biometrics/UdfpsController.java
@@ -320,19 +320,6 @@ public class UdfpsController implements DozeReceiver {
                 mView.setDebugMessage(message);
             });
         }
-
-        @Override
-        public void onAcquired(int sensorId, int acquiredInfo, int vendorCode) {
-            mFgExecutor.execute(() -> {
-                if (acquiredInfo == 6 && (mStatusBarStateController.isDozing() || !mScreenOn)) {
-                    if (vendorCode == 22) { // Use overlay to determine pressed vendor code?
-                        mPowerManager.wakeUp(mSystemClock.uptimeMillis(),
-                                PowerManager.WAKE_REASON_GESTURE, TAG);
-                        onAodInterrupt(0, 0, 0, 0); // To-Do pass proper values
-                    }
-                }
-            });
-        }
     }
 
     /**
diff --git a/services/core/java/com/android/server/biometrics/sensors/fingerprint/hidl/FingerprintAuthenticationClient.java b/services/core/java/com/android/server/biometrics/sensors/fingerprint/hidl/FingerprintAuthenticationClient.java
index c2d2e813cc1e..3058e2508f5f 100644
--- a/services/core/java/com/android/server/biometrics/sensors/fingerprint/hidl/FingerprintAuthenticationClient.java
+++ b/services/core/java/com/android/server/biometrics/sensors/fingerprint/hidl/FingerprintAuthenticationClient.java
@@ -101,15 +101,6 @@ class FingerprintAuthenticationClient extends AuthenticationClient<IBiometricsFi
         return new CompositeCallback(mALSProbeCallback, callback);
     }
 
-    @Override
-    public void onAcquired(int acquiredInfo, int vendorCode) {
-        super.onAcquired(acquiredInfo, vendorCode);
-        try {
-            mSensorOverlays.ifUdfps(controller -> controller.onAcquired(getSensorId(), acquiredInfo, vendorCode));
-        } catch (Exception e) {
-        }
-    }
-
     @Override
     public void onAuthenticated(BiometricAuthenticator.Identifier identifier,
             boolean authenticated, ArrayList<Byte> token) {
-- 
2.32.0 (Apple Git-132)

